test/embedded/simd.swift

```diff
@@ -0,0 +1,29 @@
+// RUN: %empty-directory(%t)
+
+// RUN: %target-swift-frontend %s %S/Inputs/print.swift -parse-as-library -enable-experimental-feature Embedded -c -o %t/main.o
+// RUN: %target-clang %t/main.o -o %t/a.out -dead_strip
+// RUN: %target-run %t/a.out | %FileCheck %s
+
+// REQUIRES: swift_in_compiler
+// REQUIRES: executable_test
+// REQUIRES: optimized_stdlib
+// REQUIRES: VENDOR=apple
+// REQUIRES: OS=macosx
```

Comment:
Does this test run on Linux too? Can you drop the VENDOR+OS requirement in favor of `// REQUIRES: OS=macosx || OS=linux-gnu` like the other tests?

Reply:
Yeah, probably. I just copied it from another test 😅. I'll change it.